README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/neomake/neomake.svg?branch=master)](https://travis-ci.org/neomake/neomake)
 
-# Neomake
+#[![Neomake](https://cloud.githubusercontent.com/assets/111942/21837263/b47caf4e-d798-11e6-989c-393699da9637.png)](#neomake)
```

Comment:
Didn't know if you wanted the build badge to be above or below.  So, I just replaced the header text.